File: t/PACSFramework/VB/StorageServerManager/UI/AdministrativeSettings/PasswordOptions/ActivateIdleMonitorEventArgs.vb
' *************************************************************
' Copyright (c) 1991-2019 LEAD Technologies, Inc.              
' All Rights Reserved.                                         
' *************************************************************
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text

Namespace Leadtools.Demos.StorageServer.UI.AdministrativeSettings
   Public Class ActivateIdleMonitorEventArgs : Inherits EventArgs
      Private _Activate As Boolean

      Public Property Activate() As Boolean
         Get
            Return _Activate
         End Get
         Set(ByVal value As Boolean)
            _Activate = Value
         End Set
      End Property

      Public Sub New(ByVal activate_Renamed As Boolean)
         _Activate = activate_Renamed
      End Sub
   End Class
End Namespace

File: t/PACSFramework/CS/StorageServerManager/UI/AdministrativeSettings/PasswordOptions/ActivateIdleMonitorEventArgs.cs
// *************************************************************
// Copyright (c) 1991-2019 LEAD Technologies, Inc.              
// All Rights Reserved.                                         
// *************************************************************
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Leadtools.Demos.StorageServer.UI.AdministrativeSettings
{
   public class ActivateIdleMonitorEventArgs : EventArgs
   {
      private bool _Activate;

      public bool Activate
      {
         get { return _Activate; }
         set { _Activate = value; }
      }

      public ActivateIdleMonitorEventArgs(bool activate)
      {
         _Activate = activate;
      }
   }
}
